pennylane/templates/subroutines/interferometer.py

```diff
@@ -155,7 +155,7 @@ def circuit(params):
 
         Using these random parameters, the resulting circuit is:
 
-        >>> print(qml.draw(circuit)(params))
+        >>> print(qml.draw(circuit, expansion_strategy="device")(params))
```

Comment:
What was the change in the codebase that caused this to be required?

Reply:
I think this is due to the refractor of the Interferometer template https://github.com/PennyLaneAI/pennylane/pull/1946/files